pom.xml

```diff
@@ -28,6 +29,25 @@
     <version>5.1.2-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>${project.artifactId}</name>
+    <description>Build criterion and ecosystem above multi-model databases</description>
+    
```

Comment:
Could it possible to not change this part which is for description only, but the module and dependencies are most important, it is better to keep them in the front of the pom file

Reply:
Sorry, it cannot be modified here.